### ISSUE TITLE
Browser Support?

### ISSUE THREAD

**Issue by jescalan** [emotion-js/emotion#682]
Just curious if there's a spot I wasn't able to find in the docs that lists what browsers are supported by emotion. I just deployed a component that uses it and have a torrent of errors from IE11 saying that it doesn't understand `WeakMap` - is this an oversight, recommended to be polyfilled independently, or intended not to support IE11?

Whatever the case, I'd be happy to help out with a PR if that would be useful, just want to make sure I understand the intent here!

**Comment by mitchellhamilton** [comment]
[WeakMaps are supported in IE11](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap#Browser_compatibility) so I'm not sure why you're getting errors.

We support modern evergreen browsers and IE11 (and you can polyfill WeakMap to support down to IE10 and IE9 but we don't explicitly support it)

**Comment by jescalan** [comment]
Hahah you know, that is odd. You're clearly right - I saw the errors and had not even considered this. It seems like only one or two IE11 users got this error rather than all of them, and I'm sort of baffled as to why they would be the case, so I just closed it out.

Regardless, it would be great to have a statement somewhere in the readme/docs that talked about what browsers are supported, just so everyone knows!
